lib/active_model_serializers.rb

```diff
@@ -15,6 +15,14 @@ module ActiveModelSerializers
   autoload :Adapter
   autoload :JsonPointer
   autoload :Deprecate
+  autoload :Attribute
```

Comment:
these shouldn't be here. they're serializer/resource concerns. They should be in a serializer/resource namespace